package.json

```diff
@@ -4,11 +4,11 @@
   "description": "A library that handles payload encryption for the Web Push protocol",
   "main": "src/index.js",
   "scripts": {
-    "test": "gulp lint test:manual",
+    "publish-release": "./node_modules/sw-testing-helpers/project/publish-release.sh",
```

Comment:
Can do, but I'm largely trying to discourage people from publishing-docs manually and making it an automated things, some obscurity from that P.O.V. works towards that end.